test/app/webpack.config.js

```diff
@@ -2,7 +2,7 @@ const path = require('path')
 
 module.exports = (_env, argv) => ({
   entry: './app.ts',
-  target: ['web', 'es5'],
+  target: ['node', 'es2018'],
```

Comment:
The test app used to test if the SDK works for SSR was targeting the web. I fixed it to target node and es2018.

Reply:
> The test app used to test if the SDK works for SSR was targeting the web.

Is it what is tested? As I understand it is used to test the npm setup and it's injected in a html: https://github.com/DataDog/browser-sdk/blob/7ea12b0df73b84690b7a090695b8a0c06e2ccf31/test/e2e/lib/framework/pageSetups.ts#L153

Reply:
I made two webpack config file:
- webpack.ssr.js used for the ssr test
- webpack.web.js used for e2e tests
commit: https://github.com/DataDog/browser-sdk/pull/3152/commits/1e3d8b51a184565f71a154a0d80707a140ca3393